LastWork(newWork)/css/p_login.css

```diff
@@ -0,0 +1,85 @@
+.head_section{
+	background-image: url("../Images/feedback-banner.jpg");
+	width: 100%;
+}
+
+
+header{
+	height: 640px;
+}
+
+.loginform_div{
+	background-color: #ffffff;
+	height: 360px;
+	width: 380px;
+	align-self: center;
+}
+
+.loginform_header{
+	background-color: #03a8f3;
+	text-align: center;
+	line-height: 70px;
+    box-shadow: 0px 5px 10px grey;
+}
+
+.loginform_header h1{
+	margin: 0;
+	color: #ffffff;
+}
+
+.loginform_body{
+	padding: 40px;
+	text-align: center;
+}
+
+.loginform_body_input{
+	width: 100%;
+	height: 40px;
+	margin-top: 10px;
+	border: 0;
+	border-bottom: 1px solid #828384;
+	outline: none;
+}
+
+.loginform_body_button{
+	margin-top: 40px;
+	padding: 10px 40px;
+	font-size: 15px;
+	color: #ffffff;
+	background-color: #03a8f3;
+	box-shadow: 0px 5px 10px grey;
+}
+
+.image{
+	width: 450px;
+	height: 400px;
+}
+
+p, h1{
+	margin: 70px 100px;
+}
+
+.body_section2{
+	background-color: #EFEEEC;
+}
+
+.section_height{
+	height: 640px;
+}
+
+.div_padding{
+	padding: 50px 0px;
+}
+
+.footer_section{
+	width: 100%; 
```

Comment:
segregate structure and skin. Also check if this selector isn't duplicated in other files